cypress.config.ts

```diff
@@ -27,8 +27,9 @@ module.exports = defineConfig({
   },
   e2e: {
     baseUrl: 'http://localhost:5601',
+    supportFile: 'cypress/support/e2e.{js,jsx,ts,tsx}',
     specPattern: 'cypress/integration/**/*_spec.{js,jsx,ts,tsx}',
-    testIsolation: false,
+    testIsolation: true,
```

Comment:
As a part of the Cypress I saw a couple existing tests that are not written in an isolated manner, which caused failures when I enabled this initially.

If all our tests pass with isolation on now, that's great! Let's just check the cypress tests on this PR when they run to verify that there are no new failures due to this enablement

Reply:
I don't think that the other tests are running, I will revert the testIsolation to false and change my test to reflect the lack of test isolation.

Reply:
Addressed in newest commit.